driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -201,6 +202,17 @@ public static String getDriverVersion() {
         return driverProperties.getString("driver.version");
     }
 
+    /**
+     * Logs the driver version to the console.
+     * <p>
+     * This method logs the version using the logger {@code com.datastax.driver.core}
+     * and level {@code INFO}.
+     */
+    public static void logDriverVersion() {
+        Logger core = LoggerFactory.getLogger("com.datastax.driver.core");
+        core.info("DataStax Java driver for Apache Cassandra version {}", getDriverVersion());
```

Comment:
The wording of this might be easy to misunderstand as the DataStax Java driver for: "Apache Cassandra version X".  In other words, mistaking the version logged as being for Apache Cassandra and not the java driver.  Does something like "DataStax Java driver 1.2.3 for Apache Cassandra" satisfy legal requirements?

Reply:
Very good point, I will amend, thanks! 👍